src/test.jl

```diff
@@ -0,0 +1,6 @@
+include("OHDSIAPI.jl")
+using .OHDSIAPI
```

Comment:
So, could you read through this section on testing? https://docs.julialang.org/en/v1/stdlib/Test/#Workflow-for-Testing-Packages I think it will help with your workflow here :smile: